CloudNotes/CloudNotes/MemoDetailView/MemoDetailViewController.swift

```diff
@@ -10,30 +10,49 @@ import UIKit
 class MemoDetailViewController: UIViewController {
 
     private let textView = UITextView()
+    private var textViewBottomAnchor: NSLayoutConstraint?
+    private var textViewHeightAnchor: NSLayoutConstraint?
+    private var sendingDataToListViewController: DispatchWorkItem?
+
+    private var isCreatingNewMemo = false {
+        didSet {
+            if isCreatingNewMemo == true {
+                navigationItem.rightBarButtonItem?.isEnabled = false
+            } else {
+                navigationItem.rightBarButtonItem?.isEnabled = true
+            }
```

Comment:
`새로운 메모를 만드는 중인지 아닌지` 이라는 의미인가요? 아니면 `새로운 메모를 만들었는지 아닌지`라는 의미일까요? 후자라면 네이밍을 잘못한듯 합니다.
isCreatingNewMemo 자체가 Bool 타입이라 바로 if문 컨디션에 넣어도 될거같습니다.
```suggestion
            if isCreatingNewMemo {
                navigationItem.rightBarButtonItem?.isEnabled = false
            } else {
                navigationItem.rightBarButtonItem?.isEnabled = true
            }
```

또는 아래와 같이 바꿔볼수도 있겠네요.

```suggestion
            navigationItem.rightBarButtonItem?.isEnabled = !isCreatingNewMemo
```